test/functional/page_objects/discover_page.ts

```diff
@@ -84,8 +82,10 @@ export function DiscoverPageProvider({ getService, getPageObjects }: FtrProvider
     }
 
     public async waitUntilSearchingHasFinished() {
-      const spinner = await testSubjects.find('loadingSpinner');
-      await find.waitForElementHidden(spinner, defaultFindTimeout * 10);
+      await retry.waitFor('loading spinner doesnt exist', async () => {
```

Comment:
Would it be possible to use `testSubjects.missingOrFail` here?

Reply:
@flash1293 wouldn't that fail if there was a request that would need a bit longer than expected (longer than the `defaultFindTimeout`), and the loading spinner would'n disappear ?

Reply:
I didn't test, looking at the code I thought it would have a timeout as well, but not necessarily worth investigating

Reply:
it has, and since it's simpler that way, I've change it, thx 👍